### DISCUSSION TITLE
Celebrating 10+ Years of Wired!

### DISCUSSION THREAD

**Discussion by hlj24** [thegetty/quire#91] [Show & Tell]
Duke University's Wired! Lab for Digital Art History and Visual Culture recently celebrated our 10-year anniversary. As part of our celebrations, we authored a report, ["Celebrating 10+ Years of Wired!"](https://reports.dahvc.org/), and decided to host the digital version using Quire.

**Comment by geealbers** [comment]
Thanks for sharing @hlj24! It turned out great, love all the video embeds, and as a long-time Wired! Lab admirer myself, I'm tickled to have the report published with Quire. 😊
